index.html

```diff
@@ -39,257 +42,263 @@
+          <a class="product" href="product.html">
+            <div class="label-new">NEW</div>
             <img src="img/product-4.jpg" width="184" height="164" alt="Перфоратор BOSCH BFG 2000">
-            <h3>Перфоратор BOSCH BFG 3000</h3>
-            <p>12500 Р.</p>
+            <h3 class="product-name">Перфоратор BOSCH BFG 2000</h3>
+            <p class="button price">12500 Р.</p>
           </a>
         </li>
       </ul>
 
     </section>
     <section>
-      <h2>популярные производители:</h2>
-      <p><a href="">все производители</a></p>
-      <ul>
-        <li>
+      <div class="popular">
+        <h2>популярные производители:</h2>
+        <a class="button producer-button" href="">все производители</a>
+      </div>
+      <ul class="producers">
+        <li class="producer">
           <a href="">
             <img src="img/producer-1.jpg" width="220" height="145" alt="BOSCH">
           </a>
         </li>
-        <li>
+        <li class="producer">
           <a href="">
             <img src="img/producer-2.jpg" width="220" height="145" alt="Makita">
           </a>
         </li>
-        <li>
+        <li class="producer">
           <a href="">
             <img src="img/producer-4.jpg" width="220" height="145" alt="DeWALT">
           </a>
         </li>
-        <li>
+        <li class="producer">
           <a href="">
             <img src="img/producer-5.jpg" width="220" height="145" alt="интерскол">
           </a>
         </li>
-        <li>
+        <li class="producer">
           <a href="">
             <img src="img/producer-6.jpg" width="220" height="145" alt="HITACHI">
           </a>
         </li>
-        <li>
+        <li class="producer">
           <a href="">
             <img src="img/producer-7.jpg" width="220" height="145" alt="LG">
           </a>
         </li>
-        <li>
+        <li class="producer">
           <a href="">
             <img src="img/producer-8.jpg" width="220" height="145" alt="AEG">
           </a>
         </li>
-        <li>
+        <li class="producer">
           <a href="">
             <img src="img/producer-9.jpg" width="220" height="145" alt="metabo">
           </a>
         </li>
       </ul>
     </section>
-    <section>
+    <section class="services">
       <h2>сервисы</h2>
-      <p>Хороший интернет-магазин отличается от плохого не только ценами!
+      <p class="service-description">Хороший интернет-магазин отличается от плохого не только ценами!
         Мы стараемся изо всех сил, чтобы сделать ваши покупки приятными.</p>
-      <form action="https://echo.htmlacademy.ru/courses" method="post">
-        <p><label><input type="radio" name="service" checked>Доставка</label></p>
-        <p><label><input type="radio" name="service">Гарантия</label></p>
-        <p><label><input type="radio" name="service">Кредит</label></p>
-      </form>
+      <p><label class="service-button service-button-checked"><input type="radio" name="service"
```

Comment:
Лучше так
```suggestion
      <p><label class="service-radio-input"><input type="radio" name="service"
```
Стилизацию будем делать через псевдокласс `:checked` от инпутов

Reply:
Попробовала сделать с помощью :checked. 
Пробовала разные варианты: 
input[type="radio"]:checked, 
input:checked, 
.service-radio-input:checked.
Ничего из этого не сработало почему-то.

Reply:
Не сработало потому, что мы будем стилизовать не сам `input`, его лейбл.
Для этого нужно:
- Поставить `input` перед `label`, связать их через атрибут `for`.
- Стилизовать через соседствующий селектор: `input[type="radio"]:checked + label`

[Референс по сложным селекторам](https://learn.shayhowe.com/advanced-html-css/complex-selectors/#sibling-selectors)

Reply:
Извини, тут я имел ввиду переименовать не в `service-radio-input`, а в `service-radio-label`. Бес попутал

Reply:
Сделала с помощью input[type="radio"]:checked + label. 9a7e1e1b424454f47c391bb208c079bc513574c4